Source/AlchemyVisionV1/AlchemyVision.swift

```diff
@@ -178,8 +178,8 @@ public class AlchemyVision {
      - parameter success: A function executed with information about the identified primary image.
      */
     public func getImage(
-        html: String,
-        url: String? = nil,
+        fromHTML html: String,
+        withURL url: String? = nil,
         failure: ((Error) -> Void)? = nil,
```

Comment:
Wasn't entirely sure how to approach this. We could name it `fromHTMLString` to match the `fromHTMLFile` above, but that's not very descriptive. `HTMLString` doesn't really make much sense.